storagedriver/s3/s3.go

```diff
@@ -642,6 +642,15 @@ func (d *Driver) URLFor(path string, options map[string]interface{}) (string, er
 		return "", storagedriver.InvalidPathError{Path: path}
 	}
 
+	methodString := "GET"
+	method, ok := options["method"]
+	if ok {
+		if method != "GET" && method != "HEAD" {
+			return "", storagedriver.ErrUnsupportedMethod
+		}
+		methodString, _ = method.(string)
```

Comment:
It will not, because you will never reach the line if method is not either the string "GET" or teh string "HEAD"